src/main/java/htsjdk/samtools/SamReaderFactory.java

```diff
@@ -74,11 +76,13 @@
 public abstract class SamReaderFactory {
 
     private static ValidationStringency defaultValidationStringency = ValidationStringency.DEFAULT_STRINGENCY;
-    
+
+    protected Function<SeekableByteChannel, SeekableByteChannel> pathWrapper;
```

Comment:
If we're going to include this in the base class with accessors, lets just make it private.